index.js

```diff
@@ -52,10 +65,19 @@ generateOutput(
   mappingsMetadata
 );
 
+
+var aux = {"aux": getInfoForIndex(options,rawMetadata), "notImplemented": tsAPIMetadata, "validate1": validate1(options,rawMetadata), "validate2": validate2(options)}
+
+generateOutput(
+  options["indexOutputFilePath"],
+  webStandardFunctionsIndex,
+  aux
+);
+
 //////// Helper functions
 
 function generateOutput(outFileRelPath, generator, metadata) {
-  const outFilePath = absolutePath(outFileRelPath);
+  const outFilePath = /* absolutePath( */outFileRelPath/* ) */;
```

Comment:
Should avoid commented implementations (source control is a better to see changes)